spec/abstractops.html

```diff
@@ -860,11 +860,8 @@ <h1>ISO 8601 grammar</h1>
       FractionalPart :
           DecimalDigit DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit? DecimalDigit?
 
-      TimeFractionalPart :
```

Comment:
Will there be similar removals of e.g. TimeZoneUTCOffsetFractionalPart (and FractionalPart itself, for that matter)?

Reply:
That seems like a good thing to do, but it would be better as a separate editorial change.

Reply:
→ #1985